changes/change_gate_renames.md

```diff
@@ -0,0 +1,4 @@
+Renamed QC gates:
+* Informatics Review -> Analysis Review
+* Draft Report -> Release Approval
+* Final Report -> Release
```

Comment:
(A small but important terminology note: these are updates to QC signoff names, not QC gate names! The final two QC gates remain "Informatics Pipelines + Variant Interpretation" and "Final Report")

Reply:
(So could the changelog please be updated to indicate that these are renamed QC signoffs?)

Reply:
A QC gate is a set of requirements to proceed beyond a point in the case. The sign-off is a declaration of those requirements having been met (or not). I don't think it makes sense to separate them. If they were separate, does that mean the "Informatics Pipelines + Variant Interpretation" metrics should be displayed somewhere separately from the "Analysis Review" sign-off?

Reply:
There can be multiple QC signoffs within a single gate - the current "Infomatics Review" and "Draft Report" QC signoffs are separate signoffs for the "Informatics Pipelines + Variant Interpretation" QC gate.

Reply:
<img width="728" alt="image" src="https://github.com/oicr-gsi/cardea/assets/12240540/50af8c0d-753d-455c-8f49-76d3f5192fc0">

Do these headings in Dimsum not represent QC gates? Each column may include several items along with their metrics, and multiple sign-offs per item.

The Full-Depth Sequencing gate has 2 sign-offs, called QC status, and data review, which are represented inside of the single "Full-Depth Sequencing" QC gate column. Those are sign-offs from two different people for the same task, i.e. two sign-offs for the same gate. "Release Approval" and "Release" sign-offs are from two different people for two different tasks, similar to Extraction and Library Prep.

The main point of this renaming was so that these headings would no longer be specific to clinical. Doesn't that mean we want to rename the gates?